src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -50,11 +50,12 @@
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 import static uk.gov.pay.adminusers.service.AdminUsersExceptions.internalServerError;
 
-@Path(InviteResource.INVITES_RESOURCE)
+@Path("/")
 @Tag(name = "Invites")
 public class InviteResource {
 
-    public static final String INVITES_RESOURCE = "/v1/api/invites";
+    public static final String INVITES_RESOURCE_V1 = "/v1/api/invites";
+    public static final String INVITES_RESOURCE_V2 = "/v2/api/invites";
```

Comment:
Can we remove these constants and in-line all the usages. This is something we've gradually done in other apps, so would be good to make a start here.